tests/js/spec/views/performance/data.spec.jsx

```diff
@@ -90,4 +90,72 @@ describe('generatePerformanceEventView()', function () {
       expect.stringContaining('event.type:transaction')
     );
   });
+
+  it('gets the right column', function () {
+    const result = generatePerformanceEventView(organization, {
+      query: {
+        query: 'key:value tag:value',
+      },
+    });
+    expect(result.fields).toEqual(
+      expect.arrayContaining([expect.objectContaining({field: 'user_misery(300)'})])
+    );
+    expect(result.fields).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({field: 'count_miserable(user,300)'}),
+      ])
+    );
+    expect(result.fields).toEqual(
+      expect.arrayContaining([expect.objectContaining({field: 'apdex(300)'})])
+    );
+
+    expect(result.fields).not.toEqual(
+      expect.arrayContaining([expect.objectContaining({field: 'uuser_misery_new()'})])
+    );
+    expect(result.fields).not.toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({field: 'count_miserable_new(user)'}),
+      ])
+    );
+    expect(result.fields).not.toEqual(
+      expect.arrayContaining([expect.objectContaining({field: 'apdex_new()'})])
+    );
+
+    const newOrganization = TestStubs.Organization({
+      apdexThreshold: 300,
```

Comment:
Is apdex also 300 by default? I'd maybe change this to some other number to ensure the default isn't still being used? Minor though.